components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/SessionMgtDBQueries.java

```diff
@@ -0,0 +1,121 @@
+            "SESSION_METADATA AS (" + LOAD_SESSION_METADATA_DATA + "), APPS AS (" + LOAD_APP_DATA + "), " +
+            "SESSION_APPS AS (" + LOAD_SESSION_APP_DATA_POSTGRES + ") SELECT ss.SESSION_ID, TIME_CREATED, USER_ID, " +
+            "IP_ADDRESS, LOGIN_TIME, LAST_ACCESS_TIME, USER_AGENT, APPLICATIONS FROM IDN_AUTH_SESSION_STORE ss " +
+            "JOIN IDN_AUTH_USER_SESSION_MAPPING sm ON sm.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_METADATA md ON md.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_APPS sa ON sa.SESSION_ID = ss.SESSION_ID " +
+            "WHERE (ss.SESSION_ID, TIME_CREATED) IN (SELECT SESSION_ID, TIME_CREATED FROM SESSIONS) AND " +
+            "OPERATION = ''STORE'' AND SESSION_TYPE = ''AppAuthFrameworkSessionContextCache''{3} " +
+            "ORDER BY TIME_CREATED {4} LIMIT {5}";
+
+    public static final String LOAD_SESSIONS_DB2 = "WITH SESSIONS AS (" + LOAD_SESSION_STORE_DATA + "), " +
+            "SESSION_METADATA AS (" + LOAD_SESSION_METADATA_DATA + "), APPS AS (" + LOAD_APP_DATA + "), " +
+            "SESSION_APPS AS (" + LOAD_SESSION_APP_DATA_DB2 + ") SELECT ss.SESSION_ID, TIME_CREATED, USER_ID, " +
+            "IP_ADDRESS, LOGIN_TIME, LAST_ACCESS_TIME, USER_AGENT, APPLICATIONS FROM IDN_AUTH_SESSION_STORE ss " +
+            "JOIN IDN_AUTH_USER_SESSION_MAPPING sm ON sm.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_METADATA md ON md.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_APPS sa ON sa.SESSION_ID = ss.SESSION_ID " +
+            "WHERE (ss.SESSION_ID, TIME_CREATED) IN (SELECT SESSION_ID, TIME_CREATED FROM SESSIONS) AND " +
+            "OPERATION = ''STORE'' AND SESSION_TYPE = ''AppAuthFrameworkSessionContextCache''{3} " +
+            "ORDER BY TIME_CREATED {4} FETCH FIRST {5} ROWS ONLY";
+
+    public static final String LOAD_SESSIONS_ORACLE = "WITH SESSIONS AS (" + LOAD_SESSION_STORE_DATA + "), " +
+            "SESSION_METADATA AS (" + LOAD_SESSION_METADATA_DATA + "), APPS AS (" + LOAD_APP_DATA + "), " +
+            "SESSION_APPS AS (" + LOAD_SESSION_APP_DATA_ORACLE + ") SELECT ss.SESSION_ID, TIME_CREATED, USER_ID, " +
+            "IP_ADDRESS, LOGIN_TIME, LAST_ACCESS_TIME, USER_AGENT, APPLICATIONS FROM IDN_AUTH_SESSION_STORE ss " +
+            "JOIN IDN_AUTH_USER_SESSION_MAPPING sm ON sm.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_METADATA md ON md.SESSION_ID = ss.SESSION_ID " +
+            "JOIN SESSION_APPS sa ON sa.SESSION_ID = ss.SESSION_ID " +
+            "WHERE (ss.SESSION_ID, TIME_CREATED) IN (SELECT SESSION_ID, TIME_CREATED FROM SESSIONS) AND " +
+            "OPERATION = ''STORE'' AND SESSION_TYPE = ''AppAuthFrameworkSessionContextCache''{3} " +
+            "ORDER BY TIME_CREATED {4} FETCH FIRST {5} ROWS ONLY";
+
+    public static final String LOAD_SESSIONS_MSSQL = "WITH SESSIONS AS (" + LOAD_SESSION_STORE_DATA + "), " +
```

Comment:
Please check if the queries are working properly in the relevant databases and in all supported versions (if not already done)

Reply:
Tested on H2, MySQL 5.7/8.0, Oracle 12c/19c, Postgres 10.5/12.2/13, DB2 11.5, SQLServer 2017/2019.